src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/insertfield/insertfield.html

```diff
@@ -33,10 +33,9 @@
 
                     <!-- Default value -->
                     <div>
-                        <div style="margin-bottom: 20px;">
-                            <i class="icon icon-add blue" ng-hide="vm.showDefaultValue"></i>
-                            <a href="" ng-click="vm.showDefaultValue=true" ng-hide="vm.showDefaultValue"><localize key="templateEditor_addDefaultValue">Add default value</localize></a>
-                        </div>
+                        <a prevent-default href="" class="umb-node-preview-add mb3" ng-click="vm.showDefaultValue=true" ng-hide="vm.showDefaultValue">
```

Comment:
Should this be a `<button>` instead, since it is not used as an actual anchor link?

Reply:
It probably could be.. it has an empty href and this is how most of add for picker with `umb-node-preview-add` works.
https://github.com/umbraco/Umbraco-CMS/search?q=umb-node-preview-add&unscoped_q=umb-node-preview-add

If we should change this, we should probably change it many other places as well :)

Reply:
@bjarnef - quite a few nice accessibility PRs you could put in then in the future! 

Make sure to update this one for now, our visually impaired users will thank you for it! 👍

Reply:
@bjarnef I think you can also just give it `role="button"`, that should be enough for now, see (for example): https://github.com/umbraco/Umbraco-CMS/pull/6555/files

Reply:
@nul800sebastiaan it seems updating these `add` buttons to use `button` element is more correct. Maybe @BatJan want to submit a PR to update this in the different parts of the backoffice UI?

Reply:
@bjarnef I was wrong and I've been told it needs to be a proper button! Could you update that so that @BatJan doesn't have to go in and correct it again? :-)

Reply:
@nul800sebastiaan I can do that, but shouldn't we submit a different PR to update this in all another parts of the UI where it also use this format? E.g. here:
https://github.com/umbraco/Umbraco-CMS/blob/v8/dev/src/Umbraco.Web.UI.Client/src/views/content/protect.html#L50

Just do a search for `umb-node-preview-add` 😁

Reply:
@bjarnef I love your enthusiasm to fix all the things, but if we do it in small steps then it's much easier to handle! So I would like to keep this small and then move on to the next area. Make sense?

Reply:
@nul800sebastiaan sure, I can update this anchor to a `button` element, but it wasn't really was this PR is about :)
And it just use the same markup as in this rest of the code. So once this was merged, it would be easier to submit a new PR to change the `<a prevent-default href="" class="umb-node-preview-add">` to `<button class="umb-node-preview-add">`

Reply:
@nul800sebastiaan all fixed now 👍 🌮